sapl/base/forms.py

```diff
@@ -653,14 +659,29 @@ class Meta:
                   'proposicao_incorporacao_obrigatoria',
                   'cronometro_discurso',
                   'cronometro_aparte',
-                  'cronometro_ordem']
+                  'cronometro_ordem',
+                  'mostrar_brasao_painel']
 
     def __init__(self, *args, **kwargs):
         super(ConfiguracoesAppForm, self).__init__(*args, **kwargs)
         self.fields['cronometro_discurso'].widget.attrs['class'] = 'cronometro'
         self.fields['cronometro_aparte'].widget.attrs['class'] = 'cronometro'
         self.fields['cronometro_ordem'].widget.attrs['class'] = 'cronometro'
 
+    def clean_mostrar_brasao_painel(self):
+        mostrar_brasao_painel = self.cleaned_data.get('mostrar_brasao_painel',
+                                                                        False)
+        casa = CasaLegislativa.objects.first()
+
+        if not casa:
+            raise ValidationError("Não há casa legislativa relacionada")
+
+        if (not bool(casa.logotipo) and mostrar_brasao_painel):
```

Comment:
não precisa do `bool(casa.logotipo)`, o python aceita `if not casa.logotipo and ...`